decomp/semantics/uds.py

```diff
@@ -181,71 +214,128 @@ def from_conll(cls,
                 raise ValueError('Unrecognized annotation type {0} '\
                                  'for annotation {1}.'.format(ann_type, ann_path))
 
-        return cls(predpatt_graphs, processed_annotations)
+        return cls(predpatt_graphs, None, processed_annotations)
```

Comment:
Calling the constructor without actually passing the documents (along with the sentence-level graphs and the annotations) feels kind of weird to me, but is necessary given the way document and sentence IDs are currently read in: Since both are technically annotations (read from `ud_ids.json`), it seemed to me like we have to add the annotation to the graphs first and then construct the documents. Of course, if we choose to get rid of `ud_ids.json`, we probably wouldn't have to do this.

Reply:
An alternative that would not require us to extract the filenames from the CONLL (discussed in [this comment](https://github.com/decompositional-semantics-initiative/decomp/pull/11#issuecomment-665909170)) is to have the `uds_ids.json` live at the top level of [`decomp/decomp/data/`](https://github.com/decompositional-semantics-initiative/decomp/tree/master/decomp/data) (next to `LICENSE`), signaling that it is explicitly not an annotation. It could then be handled separately from the other annotations, loaded in as early as we need it by hard-coding a (relative) path to it into the initializer.

Reply:
Mm, that's an appealing approach. I'll make that change.